src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/index.jsx

```diff
@@ -201,7 +204,7 @@ function ConfirmedAppointmentDetailsPage({
           className="vads-u-display--block vads-u-margin-bottom--2"
           backgroundOnly
         >
-          This appointment has been canceled
+          {`${canceler} canceled this appointment.`}
```

Comment:
I didn't catch this before, but we might want to guard against the facility info load taking some time to load or failing. Before the facility info is loaded and we have a facility name this might say "undefined canceled this appointment".

Reply:
👍 Added a default `'Facility'` string if there are any issues loading facility data.